droidlet/dashboard/web/src/MobileMainPane.js

```diff
@@ -10,17 +10,26 @@ import NavbarComponent from "./components/NavbarComponent";
 import MobileHomePane from "./components/MobileHomePane";
 import MobileNavigationPane from "./components/MobileNavigationPane";
 import MobileSettingsPane from "./components/MobileSettingsPane";
-
+import ObjectFixup from "./components/ObjectFixup";
+import stateManager from "./StateManager";
 class MobileMainPane extends React.Component {
   constructor(props) {
     let width = window.innerWidth;
     super(props);
+    // screen is the current pane
+    // width is the size of the feeds
+    // object_rgb is the object feed that gest feed into annotation pane
     this.state = {
       screen: "home",
       imageWidth: width / 2 - 25,
```

Comment:
You can probably make this a percentage instead and it would translate to other screen sizes